apps/rush-lib/src/logic/ChangeFiles.ts

```diff
@@ -26,13 +27,23 @@ export class ChangeFiles {
    */
   public static validate(
     newChangeFilePaths: string[],
-    changedPackages: string[]
+    changedPackages: string[],
+    rushConfiguration: RushConfiguration
   ): void {
     const projectsWithChangeDescriptions: Set<string> = new Set<string>();
     newChangeFilePaths.forEach((filePath) => {
       console.log(`Found change file: ${filePath}`);
 
       const changeFile: IChangeInfo = JsonFile.load(filePath);
+
+      if (rushConfiguration.hotfixChangeEnabled) {
```

Comment:
>hotfixChangeEnabled [](start = 28, length = 19)

Because we're changing the implementation of this a bit, the docs should be updated [here](https://github.com/microsoft/web-build-tools/blob/7ba42b79254fa360c9952c5ff301dd7add33a1e0/apps/rush-lib/src/schemas/rush.schema.json#L48-L51), [here](https://github.com/microsoft/web-build-tools/blob/7ba42b79254fa360c9952c5ff301dd7add33a1e0/apps/rush-lib/assets/rush-init/rush.json#L280-L282), and [here](https://github.com/microsoft/web-build-tools/blob/9cf4692209f0dc090dbb80364817e5c898fcf9f8/rush.json#L276-L278).